GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GitFilesTests.cs

```diff
@@ -51,11 +54,12 @@ public void CreateFileInFolderTest()
 
             this.Enlistment.WaitForBackgroundOperations().ShouldEqual(true, "Background operations failed to complete.");
 
-            GVFSHelpers.ModifiedPathsShouldContain(this.fileSystem, this.Enlistment.DotGVFSRoot, folderName + "/" + Environment.NewLine);

```

Comment:
Consider renaming `ModifiedPathsShouldContain` to `ModifiedPathsShouldContainLine`, and move the newline inside the method. That way every caller doesn't have to know this detaill

Reply:
I like that idea, that will also allow for making `ModifiedPathsNewLine` private (as mentioned in a separate comment)

Reply:
I will move the newline inside the method but leave the method named `ModifiedPathsShouldContain`.  

I found `ModifiedPathsShouldContainLine` looked a bit off as the caller is not passing in lines, it's passing in the paths that ``ModifiedPaths` should (or should not) have.